gym/lib/gym/runner.rb

```diff
@@ -16,10 +16,10 @@ def run
 
       if Gym.project.ios? || Gym.project.tvos?
         fix_generic_archive # See https://github.com/fastlane/fastlane/pull/4325
-        package_app if Gym.project.produces_archive?
+        package_app
         fix_package
         compress_and_move_dsym
-        path = move_ipa if Gym.project.produces_archive?
+        path = move_ipa
```

Comment:
Should we also remove or comment out the `if Gym.project.library? || Gym.project.framework?` part?